introduction.md

```diff
@@ -0,0 +1 @@
+Here's a change for you. I hope this gets the system to recognize that I'm making a change.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.